### ISSUE TITLE
softmax model in LabelAccuracyEvaluator.py

### ISSUE THREAD

**Issue by SylviaWang1997** [UKPLab/sentence-transformers#302]
Hi,
  Thanks for sharing your amazing work!
  My dataset will be in the form of  [sentence 1] | [sentence 2] | label(either 0 or 1) which is a binary classification problem.
  I want to fine tune your model with my dataset for label prediction.
  I read though closely on training_nli.py and LabelAccuracyEvaluator.py and found that if I want to evaluate my model accuracy on classification task, I will have to use LabelAccuracyEvaluator class as my evaluator (in training_nli, you use EmbeddingSimilarityEvaluator class as evaluator on stsb dataset because it's regression evaluation).

   But when I dive deep into LabelAccuracyEvaluator.py, I found it requires a Softmax mode which I have no idea on where to find.

Code snippet in LabelAccuracyEvaluator.py:

 for step, batch in enumerate(tqdm(self.dataloader, desc="Evaluating")):
            features, label_ids = batch_to_device(batch, self.device)
            with torch.no_grad():
                _, prediction = self.softmax_model(features, labels=None)

            total += prediction.size(0)
            correct += torch.argmax(prediction, dim=1).eq(label_ids).sum().item()
        accuracy = correct/total

 Any suggestions on how I can get a softmax_model?
Can I use SoftmaxLoss class defined in the losses folder?

Thanks!

**Comment by nreimers** [comment]
See this script:
https://github.com/UKPLab/sentence-transformers/blob/master/examples/training_transformers/training_nli.py

You would pass the model stored in train_loss  from line 56 to the LabelAccuracyEvaluator

Best
Nils Reimers

**Comment by Scanf-Myname** [comment]
Hi, I have the same problem, and I found the script in https://github.com/UKPLab/sentence-transformers/blob/master/examples/training_transformers/training_nli.py is 404 not found. so I don't know how to pass a correct softmax model
Best
jiapeng li
